docs/localization.rst

```diff
@@ -6,9 +6,14 @@ Kuma is localized with `gettext <http://www.gnu.org/software/gettext/>`_.
 User-facing strings in the code or templates need to be marked for gettext
 localization.
 
-We use `Pontoon <https://pontoon.mozilla.org/>`_ to provide an easy interface
-to localizing these files. Localizers are also free to download the PO files
-and use whatever tool they are comfortable with.
+We use `Pontoon <https://pontoon.mozilla.org/projects/mdn/>`_ to provide an easy
+interface to localizing these files. Localizers are free to download the PO
+files from Pontoon and use whatever tool they are comfortable with to translate.
```

Comment:
This sounds like they could download them from pontoon, use whatever tool they are comfortable with to translate, then what -- send a PR on github? Maybe just remove this last sentence completely so that all we're saying is we use pontoon, go use that.